src/eradiate/spectral/ckd.py

```diff
@@ -143,6 +139,15 @@ def spectral_indices(self) -> t.Generator[CKDSpectralIndex]:
 # ------------------------------------------------------------------------------
 
 
+DEFAULT_QUAD_SPEC = {
+    "type": "fixed",
+    "params": {
+        "type": "gauss_legendre",
+        "n": 1,
+    },
+}
```

Comment:
I assume the `G16` variable is no longer used? (See above, I couldn't comment there).

Reply:
You assume correctly ;)